src/systems/userinput/bindings/windows-mixed-reality-user.js

```diff
@@ -0,0 +1,498 @@
+    {
+      src: { value: paths.device.keyboard.key(" ") },
+      dest: { value: paths.actions.thaw },
+      xform: xforms.falling
+    },
+    ...freezeBindings("left"),
+    ...freezeBindings("right"),
+    ...teleportationAndRotationBindings(),
+    {
+      src: {
+        value: paths.device.keyboard.key("m")
+      },
+      dest: {
+        value: paths.actions.muteMic
+      },
+      xform: xforms.rising
+    },
+    {
+      src: {
+        value: paths.device.keyboard.key("t")
+      },
+      dest: {
+        value: paths.actions.focusChat
+      },
+      xform: xforms.rising
+    }
+  ],
+
+  [sets.leftHandHoveringOnNothing]: [],
+  [sets.cursorHoveringOnNothing]: [],
+  [sets.rightHandHoveringOnNothing]: [],
+
+  [sets.rightHandTeleporting]: [
+    {
+      src: { value: rJoyNorth },
+      dest: { value: paths.actions.rightHand.stopTeleport },
+      xform: xforms.falling
+    }
+  ],
+
+  [sets.cursorHoveringOnUI]: [
+    {
+      src: { value: paths.device.wmr.right.button("trigger").pressed },
+      dest: { value: paths.actions.cursor.grab },
+      xform: xforms.rising
+    }
+  ],
+
+  [sets.leftHandHoveringOnInteractable]: [
+    {
+      src: { value: lGripPressed },
+      dest: { value: paths.actions.leftHand.grab },
+      xform: xforms.rising
+    }
+  ],
+  [sets.cursorHoveringOnInteractable]: [
+    {
+      src: { value: rGripPressed },
+      dest: { value: paths.actions.cursor.grab },
+      xform: xforms.rising
+    }
+  ],
+  [sets.rightHandHoveringOnInteractable]: [
+    {
+      src: { value: rGripPressed },
+      dest: { value: paths.actions.rightHand.grab },
+      xform: xforms.rising
+    }
+  ],
+
+  [sets.leftHandHoldingInteractable]: [
+    neverFrozenBinding,
+    {
+      src: { value: lGripPressed },
+      dest: { value: paths.actions.leftHand.drop },
+      xform: xforms.falling
+    }
+  ],
+  [sets.cursorHoldingInteractable]: [
+    neverFrozenBinding,
+    ...cursorModDeltaBindings(),
+    {
+      src: { value: rGripPressed },
+      dest: { value: paths.actions.cursor.drop },
+      xform: xforms.falling
+    }
+  ],
+  [sets.rightHandHoldingInteractable]: [
+    neverFrozenBinding,
+    {
+      src: { value: rGripPressed },
+      dest: { value: paths.actions.rightHand.drop },
+      xform: xforms.falling
+    }
+  ],
+
+  [sets.leftHandHoveringOnPen]: [],
+  [sets.cursorHoveringOnPen]: [],
+  [sets.rightHandHoveringOnPen]: [],
+
+  [sets.leftHandHoldingPen]: penBindings("left"),
+  [sets.cursorHoldingPen]: penBindings("right", true),
+  [sets.rightHandHoldingPen]: penBindings("right"),
+
+  [sets.leftHandHoveringOnCamera]: [],
+  [sets.cursorHoveringOnCamera]: [],
+  [sets.rightHandHoveringOnCamera]: [],
+
+  [sets.leftHandHoldingCamera]: holdingCameraBindings("left"),
+  [sets.cursorHoldingCamera]: holdingCameraBindings("right", true),
+  [sets.rightHandHoldingCamera]: holdingCameraBindings("right"),
+
+  [sets.inputFocused]: [
+    {
+      src: { value: "/device/keyboard" },
```

Comment:
This was one of the motivating factors behind why we created the priority system based on src hierarchy.